app/models/concerns/bib_record.rb

```diff
@@ -103,4 +59,14 @@ def bib
     def call
       item.dig('call', 'fields') || {}
     end
+
+    # Use Redis to store this globally once an hour
+    def item_type_mapping
+      redis = Redis.new
+      return JSON.parse(redis.get('item_type_map')) unless redis.get('item_type_map').nil?
+
+      client = SymphonyClient.new
+      redis.setex('item_type_map', 3600, client.get_item_type_map.to_json)
+      JSON.parse(redis.get('item_type_map'))
```

Comment:
```suggestion
      item_type_map = client.get_item_type_map.to_json
      redis.setex('item_type_map', 3600, item_type_map)
      JSON.parse(item_type_map)
```

@cdmo with not being so sure but would the above be better to avoid `redis.get` here? maybe it is an insignificant change, please let me know what you think?

Reply:
Yes thanks, good spot!